src/clue/app-config.json

```diff
@@ -246,6 +246,7 @@
         "defaultSeriesLegend": true,
         "connectPointsByDefault": true,
         "autoAssignAttributes": true,
+        "attributeDnD": false,
```

Comment:
I'd be inclined to use something a little more explicit, like `disableAttributeDnD` and have it default to `false` like the others. That way it's enabled by default but can be disabled with the option.